packages/@aws-cdk/cloudformation-include/test/test-templates/functions.json

```diff
@@ -0,0 +1,94 @@
+{
+  "Mappings" : {
+    "RegionMap" : {
+      "region-1" : { 
+        "HVM64" : "name1", "HVMG2" : "name2" 
+      }
+    }
+  },
+  "Resources": {
+
+    "Vpc": {
+      "Type" : "AWS::EC2::VPC",
+      "Properties" : {
+        "CidrBlock" : {
+            "Fn::Cidr": [ "10.0.0.0/24", "6", "5"]
+        }
+      }
+    },
+    "2ArgVpc": {
+      "Type" : "AWS::EC2::VPC",
+      "Properties" : {
+        "CidrBlock" : {
+          "Fn::Cidr": [ "10.0.0.0/24", "6"]
+        }
+      }
+    },
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": {
+          "Fn::FindInMap": [
+              "RegionMap",
+              "region-1",
+              "HVM64"
+          ]
+        }
+      }
+    },
+    "subnet1" : {
+      "Type" : "AWS::EC2::Subnet",
+      "Properties" : {
+        "VpcId" : { 
+          "Fn::Split": [
+              ",",
+              { "Fn::ImportValue" : "ImportedVpcId" }
+          ]
+        },
+        "CidrBlock" : "10.0.0.0/24",
+        "AvailabilityZone" : {
+          "Fn::Select" : [ 
+            "0", 
+            { 
+              "Fn::GetAZs" : "" 
+            } 
+          ]
+        }
+      }
+    },
+    
+    "subnet2" : {
+      "Type" : "AWS::EC2::Subnet",
+      "Properties" : {
+        "VpcId" : { 
+          "Fn::Select" : ["3", ["foo", "aValidVPCID", "foo2"]]
+        },
+        "CidrBlock" : "10.0.0.0/24",
+        "AvailabilityZone" : {
+          "Fn::Select" : [ 
+            "0", 
+            { 
+              "Fn::GetAZs" : "eu-west-2" 
+            } 
+          ]
+        }
+      }
+    },
+    "transformBucket": {
```

Comment:
Can you use uppercase names for logical IDs? So `"TransformBucket"` instead of `transformBucket`.